src/pages/Upload/components/UploadInfo/UploadPage/CategoryPage/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Button = ({ children, categoryBtnHandler, btnOn, id }) => {
+  return (
+    <CategoryBtn
+      onClick={() => categoryBtnHandler(id)}
+      btnOn={btnOn === id && 'active'}
```

Comment:
```suggestion
      btnOn={btnOn === id}
```
이렇게만 넘겨줘도 같은 의미로 동작하겠네요

Reply:
제출전에 함수의 조건들을 꼭 체크해봐야겠네요 감사합니다!!